.github/workflows/test.yml

```diff
@@ -55,12 +56,17 @@ jobs:
         make install-dev
     - name: Run Lint
       uses: samuelmeuli/lint-action@v1
+      if: ${{ github.event_name != 'pull_request_target' }}
```

Comment:
It would still run for forks by "push" event

Reply:
Seems odd but I think tox test also include flake8 and also weird that test from forks are skipped guess this would require and update to checkout action.